x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/bulk_create_attachments.ts

```diff
@@ -0,0 +1,523 @@
+          await bulkCreateAttachments({
+            supertest,
+            caseId: postedCase.id,
+            params: [
+              {
+                type: CommentType.user,
+                comment: 'test',
+                owner: 'securitySolutionFixture',
+              },
+              {
+                type: CommentType.user,
+                [attribute]: attribute,
+                comment: 'a comment',
+                owner: 'securitySolutionFixture',
+              },
+            ],
+            expectedHttpCode: 400,
+          });
+        }
+      });
+
+      it('400s when missing attributes for type alert', async () => {
+        const postedCase = await createCase(supertest, postCaseReq);
+
+        const allRequestAttributes = {
+          type: CommentType.alert,
+          index: 'test-index',
+          alertId: 'test-id',
+          rule: {
+            id: 'id',
+            name: 'name',
+          },
+          owner: 'securitySolutionFixture',
+        };
+
+        for (const attribute of ['alertId', 'index']) {
+          const requestAttributes = omit(attribute, allRequestAttributes);
+          await bulkCreateAttachments({
+            supertest,
+            caseId: postedCase.id,
+            params: [
+              {
+                type: CommentType.user,
+                comment: 'test',
+                owner: 'securitySolutionFixture',
+              },
+              // @ts-expect-error
+              requestAttributes,
+            ],
+            expectedHttpCode: 400,
+          });
+        }
+      });
+
+      it('400s when adding excess attributes for type alert', async () => {
+        const postedCase = await createCase(supertest, postCaseReq);
+
+        for (const attribute of ['comment']) {
+          await bulkCreateAttachments({
+            supertest,
+            caseId: postedCase.id,
+            params: [
+              {
+                type: CommentType.user,
+                comment: 'test',
+                owner: 'securitySolutionFixture',
+              },
+              {
+                type: CommentType.alert,
+                [attribute]: attribute,
+                alertId: 'test-id',
+                index: 'test-index',
+                rule: {
+                  id: 'id',
+                  name: 'name',
+                },
+                owner: 'securitySolutionFixture',
+              },
+            ],
+            expectedHttpCode: 400,
+          });
+        }
+      });
+
+      it('404s when case is missing', async () => {
+        await bulkCreateAttachments({
+          supertest,
+          caseId: 'not-exists',
+          params: [
+            {
+              type: CommentType.user,
+              comment: 'test',
+              owner: 'securitySolutionFixture',
+            },
+          ],
+          expectedHttpCode: 404,
+        });
+      });
+
+      it('400s when adding an alert to a closed case', async () => {
```

Comment:
This test is only testing the creation with 3 attachments (the ones in the utils functions). Could you please add tests for the following cases? 1 attachment, 3 attachments, 10 attachments. to cover the API more thoughtfully.